### ISSUE TITLE
User agent being blanked out when it is not overridden

### ISSUE THREAD

**Issue by junkinthetrunk** [xtools-at/Android-PWA-Wrapper#5]

**Comment by xtools-at** [comment]
did you check `Constants.java` and disabled all the settings?
```
	// User Agent tweaks
    public static boolean POSTFIX_USER_AGENT = true; // set to true to append USER_AGENT_POSTFIX to user agent
    public static boolean OVERRIDE_USER_AGENT = false; // set to true to use USER_AGENT instead of default one
    public static String USER_AGENT_POSTFIX = "AndroidApp"; // useful for identifying traffic, e.g. in Google Analytics
    public static String USER_AGENT = "Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/59.0.3071.115 Mobile Safari/537.36";
```

this is what the webview does, so make sure to set both OVERRIDE_USER_AGENT and POSTFIX_USER_AGENT to false:

```
// set User Agent
        if (Constants.OVERRIDE_USER_AGENT || Constants.POSTFIX_USER_AGENT) {
            String userAgent = "";
            if (Constants.OVERRIDE_USER_AGENT) {
                userAgent = Constants.USER_AGENT;
            }
            if (Constants.POSTFIX_USER_AGENT) {
                userAgent = userAgent + " " + Constants.USER_AGENT_POSTFIX;
            }
            webSettings.setUserAgentString(userAgent);
        }
```

**Comment by junkinthetrunk** [comment]
if only POSTFIX_USER_AGENT is set to true and OVERRIDE_USER_AGENT is set to false, the userAgent is set to "" and the postfix is appended.

**Comment by xtools-at** [comment]
> if only POSTFIX_USER_AGENT is set to true and OVERRIDE_USER_AGENT is set to false, the userAgent is set to "" and the postfix is appended.

fixed with commit [eca480e](https://github.com/xtools-at/Android-PWA-Wrapper/commit/eca480e35f1f41a8596f405583c9731da912cb8e).

Closing here, please reopen if necessary.
